lambda/storeHistoricalDataInDB.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	type TimestreamWriteClient,
+	type _Record,
+	WriteRecordsCommand,
+} from '@aws-sdk/client-timestream-write'
+import { chunkArray } from './chunkArray.js'
+
+export const storeHistoricalDataInDB =
+	({
+		tsw,
+		dbName,
+		tableName,
+	}: {
+		tsw: TimestreamWriteClient
+		dbName: string
+		tableName: string
+	}) =>
+	async (
+		records: _Record[],
+	): Promise<{ success: boolean } | { error: Error }> => {
+		const recordsToTimestream = chunkArray({
+			array: records,
+			chunkSize: 100,
+		}) as _Record[][]
```

Comment:
Implement the proper type signature for chunkArray. See above.

Reply:
Would this mean to remove 'as _Record[][]' since this is now fixed by the change in the resolved change above?

Reply:
Yes, when correctly typed, chunkArray will return the correctly typed result.